src/otx/api/usecases/exportable_code/prediction_to_annotation_converter.py

```diff
@@ -268,9 +273,10 @@ class ClassificationToAnnotationConverter(IPredictionToAnnotationConverter):
 
     Args:
         labels (LabelSchemaEntity): Label Schema containing the label info of the task
+        hierarchical_info (Dict): Info from model.hierarchical_info["cls_heads_info"]
     """
 
-    def __init__(self, label_schema: LabelSchemaEntity):
+    def __init__(self, label_schema: LabelSchemaEntity, hierarchical_info: Optional[Dict] = None):
```

Comment:
Same with above, want to represent the more explicit meaning. It could bring the confusion.

```suggestion
    def __init__(self, label_schema: LabelSchemaEntity, hierarchical_cls_heads_info: Optional[Dict] = None):
```